call_tree_tool/edgeprocessor.py

```diff
@@ -50,7 +50,13 @@ def process_line(self, line):
 
         funcrecord = self.database.get_function_by_name(funcname)
         if not funcrecord:
-            funcdata = ("unknown", funcname)
+            # check if function is in filter list (E.g. Compiler extensions to C language)
+            with open("func_filter", "r") as filtered_funcs:
```

Comment:
You mention the file func_filter - is it a new file that we add to the release? Or does the user have to provide this file? In both cases, more information is needed.

Reply:
Added info into Readme file